tests/Behat/FeatureContext.php

```diff
@@ -298,4 +298,16 @@ public function assertVisibility($element): void {
     $this->assertVisuallyVisible($node);
   }
 
+  /**
+   * Waits a while, for debugging.
+   *
+   * @param int $seconds
+   *   How long to wait.
+   *
+   * @When I wait :seconds second(s)
+   */
+  public function wait($seconds) {
```

Comment:
Do we need this method?

Reply:
I added it for debugging purposes.

Reply:
But it isn't used. So looks like it is useless for now.

Reply:
I removed this method.